<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileManager</class>
 <widget class="QWidget" name="FileManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FileManager</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButtonBack">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>31</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>31</width>
         <height>31</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Backward</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="back"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonNext">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Forward</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="forward"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>IBeamCursor</cursorShape>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="view-refresh"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_bookmarks">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="stock_add-bookmark"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_details">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Details View</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="view-list-details"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_icons">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Icons View</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="view-list-icons"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_options">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Options</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="settings"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
